apps/ybal/components/HarvestsListRow.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import IconLinkOut from '@yearn-finance/web-lib/icons/IconLinkOut';
+import {toAddress, truncateHex} from '@yearn-finance/web-lib/utils/address';
+import {STYBAL_TOKEN_ADDRESS} from '@yearn-finance/web-lib/utils/constants';
+import {formatBN, formatToNormalizedValue} from '@yearn-finance/web-lib/utils/format.bigNumber';
+import {formatAmount, formatUSD} from '@yearn-finance/web-lib/utils/format.number';
+import {formatDate} from '@yearn-finance/web-lib/utils/format.time';
+import {ImageWithFallback} from '@common/components/ImageWithFallback';
+
+import type {ReactElement} from 'react';
+import type {TYDaemonHarvests} from '@common/types/yearn';
+
+function	HarvestListRow({harvest}: {harvest: TYDaemonHarvests}): ReactElement {
+	return (
+		<div className={'yearn--table-wrapper'}>
+			<div className={'yearn--table-token-section'}>
+				<div className={'yearn--table-token-section-item'}>
+					<div className={'yearn--table-token-section-item-image'}>
+						<ImageWithFallback
+							alt={toAddress(harvest.vaultAddress) === STYBAL_TOKEN_ADDRESS ? 'st-yBal' : 'lp-yBal'}
```

Comment:
Sounds a bit too optimistic, perhaps we should check if `toAddress(harvest.vaultAddress)` is a valid address before this

Reply:
Looks good!